### DISCUSSION TITLE
System Prompts do not work

### DISCUSSION THREAD

**Discussion by DirtyKnightForVi** [open-webui/open-webui#4375] [Q&A]
1. Only set prompts for each individual conversation, and they can only take effect when certain running parameters are changed.
2. The prompts set in the AI conversation playground and in the settings/general section will not take effect.
3. This issue has existed since Version 3.10

**Comment by GrayXu** [comment]
I encountered the same issue in version 0.3.10, but it was resolved after upgrading to 0.3.11.

**Reply by DirtyKnightForVi** [reply]
but Interestingly，i still met this problem T_T

**Reply by DirtyKnightForVi** [reply]
same as [this](https://github.com/open-webui/open-webui/discussions/4381#discussion-7014017) ??

**Reply by DirtyKnightForVi** [reply]
I‘m trying back to 0.3.7, which before `Chat Controls` was added.
